### PR TITLE
Meth bug fix

### PR DESCRIPTION
**Pull request by DatBoiWeep**
- When completing the cooking process for meth, it will now be dropped on the floor if the player's inventory is full.

### CODE REVIEW

File: resources/usa_meth/sv_meth.lua [hunk 1]
```diff
@@ -18,11 +18,24 @@ AddEventHandler("methJob:methProduced", function(itemName, securityToken)
...
+	local newCoords = {
+		x = methProduced.coords.x,
+		y = methProduced.coords.y,
+		z = methProduced.coords.z
+	}
+	newCoords.x = newCoords.x + (math.random() * 0.5)
+	newCoords.y = newCoords.y + (math.random() * 0.5)
+	newCoords.z = newCoords.z - 0.85
+	methProduced.coords = newCoords
+	methProduced.coords = GetEntityCoords(GetPlayerPed(char.get("source")))
```
Comment (minipunch) [inline resources/usa_meth/sv_meth.lua:31]:
```suggestion
```

this overwrites the lines just before it, so it makes them pointless

so if u want the item to spawn in a random spot around the player and not just on the exact player coords every time, delete this line

and fyi it's the same exact thing as line 21, just called a bit differently

**Review by minipunch — CHANGES_REQUESTED** [review]

File: resources/usa_meth/sv_meth.lua (hunk 1 above)
Comment (DatBoiWeep) [reply resources/usa_meth/sv_meth.lua:31]: oh LOL

Comment (DatBoiWeep) [reply resources/usa_meth/sv_meth.lua:31]: I will fix sir

Comment (DatBoiWeep) [reply resources/usa_meth/sv_meth.lua]: Removed it 😅

**Review by minipunch — APPROVED** [review]

**Pull request merged**